release_notes.md

```
@@ -1,20 +1,40 @@
+- Removed Shizuku requirement for default folder access
+- Added settings management system:
+    - Introduced setting.json configuration file
+    - Added customizable world scan folder configuration
+    - Implemented new settings UI with folder management
+
+### 01Nov24
+
+- Completed world scan folders system:
+    - Added ability to add, edit, and remove scan folders
+    - Implemented settings persistence (save/load)
+- Fixed world loader update item make app crash
```

Markdownlint

Files should end with a single newline character

(MD047, single-trailing-newline)